modules/rosa-hcp-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,364 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-creation-and-sharing_{context}"]
+= Step One - VPC Owner: Configuring a VPC to share within your AWS organization
+
+You can share subnets within a configured VPC with another AWS user account if that account is within your current AWS organization.
+
+image::522-shared-vpc-step-1.png[]
+.Procedure
+
+. Create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console].
```

Comment:
I think we should reword this to reference the AWS docs instead of the `us-east-1` region console URL.

Reply:
Sadly, I can't link to the correction console page and make it region agnostic. I'll add a note though.

Reply:
@jneczypor https://console.aws.amazon.com/vpc